ivy.el

```diff
@@ -253,6 +253,12 @@ It is a list of (CALLER . HEIGHT).  CALLER is a caller of
 If `(minibuffer-depth)' equals this, `ivy-completing-read' will
 act as if `ivy-completing-read-handlers-alist' is empty.")
 
+(defvar counsel--search-commands '(counsel-git-grep
+                                   counsel-ag
+                                   counsel-rg
+                                   counsel-pt)
+  "counsel search commands")
```

Comment:
Please write full sentences, where the first non-keyword word is capitalised and the sentence ends in a full stop.  How about the following:
```el
"List of Counsel search commands."
```